src/anyvar/translate/variation_normalizer.py

```diff
@@ -0,0 +1,61 @@
+from typing import Dict
+import requests
+from ga4gh.vrs import models as vrs_models
+
+from . import _Translator, TranslatorConnectionException
+
+
+class VariationNormalizerRestTranslator(_Translator):
+
+    def __init__(self, endpoint_uri: str):
+        """Initialize normalizer-based translator.
+
+        :param endpoint_uri: base REST endpoint address
+        :raises TranslatorConnectionException: if endpoint doesn't respond to initial query
+        """
+        self.endpoint_base = endpoint_uri
+
+        openapi_docs = self.endpoint_base + "openapi.json"
+        resp = requests.get(openapi_docs)
+        if resp.status_code != 200:
+            raise TranslatorConnectionException(
+                f"Failed to get response from Variation Normalizer REST endpoint at {openapi_docs}"
+            )
+
+    def _send_rest_request(self, request_url: str):
+        """Emit normalization request. Broken out to enable mocking.
+
+        :param request_url: URL containing normalization request parameters
+        """
+        return requests.get(request_url)
+
+    def translate(self, var: str, **kwargs: Dict) -> vrs_models.Allele:
+        """Translate provided variation text into a normalized VRS object.
+
+        :param var: user-provided text object describing or referencing a variation.
+        :raises TranslatorConnectionException: if translation request returns error
+        """
+
+        req_url = self.endpoint_base + f"normalize?q={var}"
+        resp = self._send_rest_request(req_url)
+        if resp.status_code == 404:
+            raise TranslatorConnectionException(
+                f"Failed to get response from Variation Normalizer REST endpoint at {req_url}"
+            )
+        elif resp.status_code == 500:
+            raise TranslatorConnectionException(
+                f"Variation Normalizer REST endpoint returned server error for {var}"
+            )
+        elif resp.status_code != 200:
+            raise TranslatorConnectionException(
+                f"Variation Normalizer REST endpoint returned {resp.status_code} for {var}"
+            )
+
+        resp_json = resp.json()
+        if resp_json.get("warnings"):
+            raise Exception("TODO figure out what exception should get caught")
```

Comment:
Is this being addressed later? Might be nice to make a new issue so we don't forget about it if so